### PR TITLE
Add more information about manageSystem permission

### PR DESCRIPTION
**Pull request by mkuznyetsov**
### What does this PR do?
Add more information about manageSystem:
- list of available services
- Admin initialization

### CODE REVIEW

File: src/main/pages/user-management/permissions.adoc [hunk 1]
```diff
@@ -74,13 +74,41 @@ The following permissions are applicable to organizations:
 |===
 
 [id="super-priviliged-mode"]
-== Super Priviliged Mode
+== manageSystem permission
 
-The permission "manageSystem" can be extended to provide a super privileged mode that allows advanced actions to be performed on any resources managed by the system. A user with "manageSystem" permission is able read and stop any workspaces. To perform other actions on workspaces and organizations, the user will need to assign himself the permissions needed.
+User with `manageSystem` are able to perform various admin operations.
```
Comment (skabashnyuk) [inline src/main/pages/user-management/permissions.adoc]: A user with `manageSystem` permission

File: src/main/pages/user-management/permissions.adoc [hunk 2]
```diff
@@ -74,13 +74,41 @@ The following permissions are applicable to organizations:
...
+On Start, Che will grant this permission to the admin user, with default name `admin`.
```
Comment (skabashnyuk) [inline src/main/pages/user-management/permissions.adoc]: On che-master start or after the first login, this permission will be granted to the person configured with ```CHE_SYSTEM_ADMIN__NAME``` property.

File: src/main/pages/user-management/permissions.adoc [hunk 3]
```diff
@@ -74,13 +74,41 @@ The following permissions are applicable to organizations:
...
-By default, this mode is disabled.
+Users with `manageSystem` persmission will have access to the following services:
 
-It is possible to activate this option by configuring the `CHE_SYSTEM_SUPER_PRIVILEGED_MODE` in the `che.env` file.
+[cols=",,",options="header",]
+|===
+|Path | HTTP Method | Description
+|/resource/free/{accountId} |DELETE | Remove free resource limit for given account
+|/installer |PUT |Stop workspace
+|/logger/{name}|POST | Add new logger
```
Comment (skabashnyuk) [inline src/main/pages/user-management/permissions.adoc]: More info about logger. What, why, how?

File: src/main/pages/user-management/permissions.adoc (hunk 1 above)
Comment (mkuznyetsov) [reply src/main/pages/user-management/permissions.adoc]: rewrote that part

File: src/main/pages/user-management/permissions.adoc (hunk 2 above)
Comment (mkuznyetsov) [reply src/main/pages/user-management/permissions.adoc]: rewrote that part

File: src/main/pages/user-management/permissions.adoc (hunk 3 above)
Comment (mkuznyetsov) [reply src/main/pages/user-management/permissions.adoc]: fixed

File: src/main/pages/user-management/permissions.adoc [hunk 4]
```diff
@@ -73,14 +73,40 @@ The following permissions are applicable to organizations:
 |manageUsers |Allows creating and managing users.
 |===
 
-[id="super-priviliged-mode"]
-== Super Priviliged Mode
+All system permissions will be granted to the admin user configured with CHE_SYSTEM_ADMIN__NAME property (default `admin`).
```
Comment (skabashnyuk) [inline src/main/pages/user-management/permissions.adoc]: to the user configured

**Review by skabashnyuk — APPROVED** [review]
other looks good to me

File: src/main/pages/developer-guide/logging.adoc [hunk 5]
```diff
@@ -87,3 +87,24 @@ for example
...
+E.g., sending PUT request with following configuration
+
+```
+{
+  "name" : "org.eclipse.che.api.workspace.activity"
+  "level" : "DEBUG"
+}
+```
+
+will add logger level configuration that will set DEBUG level for loggers in `org.eclipse.che.api.workspace.activity` package
```
Comment (skabashnyuk) [inline src/main/pages/developer-guide/logging.adoc]: will change logger level

File: src/main/pages/developer-guide/logging.adoc [hunk 6]
```diff
@@ -87,3 +87,24 @@ for example
...
+
+[cols=",,",options="header",]
+|===
+|GET | logger/ - | get list of all loggers
+|GET | logger/{name} | get info about particular logger
+|POST | logger/{name} | update logger configuration
+|PUT | logger/{name} | create logger configuration
+|===
+
+E.g., sending PUT request with following configuration
```
Comment (rkratky) [inline src/main/pages/developer-guide/logging.adoc]:
```suggestion
For example, sending PUT request with following configuration
```

File: src/main/pages/developer-guide/logging.adoc (hunk 5 above)
Comment (rkratky) [inline src/main/pages/developer-guide/logging.adoc]:
```suggestion
adds logger level configuration that sets the `DEBUG` level for loggers in the `org.eclipse.che.api.workspace.activity` package.
```

File: src/main/pages/user-management/permissions.adoc [hunk 7]
```diff
@@ -73,14 +73,48 @@ The following permissions are applicable to organizations:
...
 
-[id="super-priviliged-mode"]
-== Super Priviliged Mode
-
-The permission "manageSystem" can be extended to provide a super privileged mode that allows advanced actions to be performed on any resources managed by the system. A user with "manageSystem" permission is able read and stop any workspaces. To perform other actions on workspaces and organizations, the user will need to assign himself the permissions needed.
-
-By default, this mode is disabled.
-
-It is possible to activate this option by configuring the `CHE_SYSTEM_SUPER_PRIVILEGED_MODE` in the `che.env` file.
+All system permissions will be granted to the admin user configured with CHE_SYSTEM_ADMIN__NAME property (default `admin`).
```
Comment (rkratky) [inline src/main/pages/user-management/permissions.adoc]:
```suggestion
All system permissions will be granted to the administration user configured with the pass:[`CHE_SYSTEM_ADMIN__NAME`] property (the default is `admin`).
```

File: src/main/pages/user-management/permissions.adoc [hunk 8]
```diff
@@ -73,14 +73,48 @@ The following permissions are applicable to organizations:
...
+This will happen at Che Server start, or if he is not present in Che user database, after his first login.
```
Comment (rkratky) [inline src/main/pages/user-management/permissions.adoc]:
```suggestion
This happens at Che Server start. If the user is not present in the Che user database, it happens after the user's login.
```

File: src/main/pages/user-management/permissions.adoc [hunk 9]
```diff
@@ -73,14 +73,48 @@ The following permissions are applicable to organizations:
...
+
+[id="manage-system-permission"]
+== manageSystem permission
+
+Users with `manageSystem` persmission will have access to the following services:
```
Comment (rkratky) [inline src/main/pages/user-management/permissions.adoc]:
```suggestion
Users with the `manageSystem` permission have access to the following services:
```

File: src/main/pages/user-management/permissions.adoc [hunk 10]
```diff
@@ -73,14 +73,48 @@ The following permissions are applicable to organizations:
...
+|Path | HTTP Method | Description
+|/resource/free/ |GET | Get free resource limits
+|/resource/free/{accountId} |GET | Get free resource limits for given account
+|/resource/free/{accountId} |POST | Edit free resource limit for given account
+|/resource/free/{accountId} |DELETE | Remove free resource limit for given account
+|/installer/ |POST | Add installer to the registry
+|/installer/{key} |PUT | Update installer in the registry
+|/installer/{key} |DELETE | Remove installer from the registry
+|/logger/ |GET | Get logging configurations in Che Server
+|/logger/{name}|GET | Get configurations of logger by it's name in Che Server
```
Comment (rkratky) [inline src/main/pages/user-management/permissions.adoc]:
```suggestion
|`/logger/{name}` |GET | Get configurations of logger by its name in Che Server
```

File: src/main/pages/user-management/permissions.adoc [hunk 11]
```diff
@@ -73,14 +73,48 @@ The following permissions are applicable to organizations:
...
+|/logger/{name}|PUT | Create Logging in Che Server
```
Comment (rkratky) [inline src/main/pages/user-management/permissions.adoc]:
```suggestion
|/logger/{name}|PUT | Create logging in Che Server
```

File: src/main/pages/user-management/permissions.adoc [hunk 12]
```diff
@@ -73,14 +73,48 @@ The following permissions are applicable to organizations:
...
+|/logger/{name}|POST | Edit Logging in Che Server
```
Comment (rkratky) [inline src/main/pages/user-management/permissions.adoc]:
```suggestion
|`/logger/{name}` |POST | Edit logging in Che Server
```

File: src/main/pages/user-management/permissions.adoc [hunk 13]
```diff
@@ -73,14 +73,48 @@ The following permissions are applicable to organizations:
...
+
+[id="manage-system-permission"]
+== manageSystem permission
+
+Users with `manageSystem` persmission will have access to the following services:
+
+[cols=",,",options="header",]
+|===
+|Path | HTTP Method | Description
+|/resource/free/ |GET | Get free resource limits
```
Comment (rkratky) [inline src/main/pages/user-management/permissions.adoc]:
```suggestion
|`/resource/free/` |GET | Get free resource limits
```

File: src/main/pages/user-management/permissions.adoc [hunk 14]
```diff
@@ -73,14 +73,48 @@ The following permissions are applicable to organizations:
...
+|/resource/free/{accountId} |GET | Get free resource limits for given account
```
Comment (rkratky) [inline src/main/pages/user-management/permissions.adoc]:
```suggestion
|`/resource/free/{accountId}` |GET | Get free resource limits for given account
```

File: src/main/pages/user-management/permissions.adoc [hunk 15]
```diff
@@ -73,14 +73,48 @@ The following permissions are applicable to organizations:
...
+|/resource/free/{accountId} |POST | Edit free resource limit for given account
```
Comment (rkratky) [inline src/main/pages/user-management/permissions.adoc]:
```suggestion
|`/resource/free/{accountId}` |POST | Edit free resource limit for given account
```

File: src/main/pages/user-management/permissions.adoc [hunk 16]
```diff
@@ -73,14 +73,48 @@ The following permissions are applicable to organizations:
...
+|/resource/free/{accountId} |DELETE | Remove free resource limit for given account
```
Comment (rkratky) [inline src/main/pages/user-management/permissions.adoc]:
```suggestion
|`/resource/free/{accountId}` |DELETE | Remove free resource limit for given account
```

File: src/main/pages/user-management/permissions.adoc [hunk 17]
```diff
@@ -73,14 +73,48 @@ The following permissions are applicable to organizations:
...
+|/installer/ |POST | Add installer to the registry
```
Comment (rkratky) [inline src/main/pages/user-management/permissions.adoc]:
```suggestion
|`/installer/` |POST | Add installer to the registry
```

File: src/main/pages/user-management/permissions.adoc [hunk 18]
```diff
@@ -73,14 +73,48 @@ The following permissions are applicable to organizations:
...
+|/installer/{key} |PUT | Update installer in the registry
```
Comment (rkratky) [inline src/main/pages/user-management/permissions.adoc]:
```suggestion
|`/installer/{key}` |PUT | Update installer in the registry
```

File: src/main/pages/user-management/permissions.adoc [hunk 19]
```diff
@@ -73,14 +73,48 @@ The following permissions are applicable to organizations:
...
+|/installer/{key} |DELETE | Remove installer from the registry
```
Comment (rkratky) [inline src/main/pages/user-management/permissions.adoc]:
```suggestion
|`/installer/{key}` |DELETE | Remove installer from the registry
```

File: src/main/pages/user-management/permissions.adoc [hunk 20]
```diff
@@ -73,14 +73,48 @@ The following permissions are applicable to organizations:
...
+|/logger/ |GET | Get logging configurations in Che Server
```
Comment (rkratky) [inline src/main/pages/user-management/permissions.adoc]:
```suggestion
|`/logger/` |GET | Get logging configurations in Che Server
```

File: src/main/pages/user-management/permissions.adoc [hunk 21]
```diff
@@ -73,14 +73,48 @@ The following permissions are applicable to organizations:
...
+|/logger/{name}|GET | Get configurations of logger by it's name in Che Server
+|/logger/{name}|PUT | Create Logging in Che Server
+|/logger/{name}|POST | Edit Logging in Che Server
+|/resource/{accountId}/details |GET | Get detailed information about resources for given account
```
Comment (rkratky) [inline src/main/pages/user-management/permissions.adoc]:
```suggestion
|`/resource/{accountId}/details` |GET | Get detailed information about resources for given account
```

File: src/main/pages/user-management/permissions.adoc [hunk 22]
```diff
@@ -73,14 +73,48 @@ The following permissions are applicable to organizations:
...
+|/system/stop |POST | Shutdown all system services, prepare Che to stop
```
Comment (rkratky) [inline src/main/pages/user-management/permissions.adoc]:
```suggestion
|`/system/stop` |POST | Shutdown all system services, prepare Che to stop
```

File: src/main/pages/user-management/permissions.adoc [hunk 23]
```diff
@@ -73,14 +73,48 @@ The following permissions are applicable to organizations:
...
+|/stacks |All methods | All Stack service methods
```
Comment (rkratky) [inline src/main/pages/user-management/permissions.adoc]:
```suggestion
|`/stacks` |All methods | All Stack service methods
```

File: src/main/pages/user-management/permissions.adoc [hunk 24]
```diff
@@ -73,14 +73,48 @@ The following permissions are applicable to organizations:
...
+|===
+
+A Super Privileged mode can be enabled to extend with additional services.
```
Comment (rkratky) [inline src/main/pages/user-management/permissions.adoc]: To extend what?

```suggestion
A super-privileged mode can be enabled to extend with additional services.
```

File: src/main/pages/user-management/permissions.adoc [hunk 25]
```diff
@@ -73,14 +73,48 @@ The following permissions are applicable to organizations:
...
+To enable it, configure environment variable `CHE_SYSTEM_SUPER_PRIVILEGED_MODE=true` in the `che.env` file.
```
Comment (rkratky) [inline src/main/pages/user-management/permissions.adoc]:
```suggestion
To enable it, configure the `CHE_SYSTEM_SUPER_PRIVILEGED_MODE=true` environment variable in the `che.env` file.
```

File: src/main/pages/user-management/permissions.adoc [hunk 26]
```diff
@@ -73,14 +73,48 @@ The following permissions are applicable to organizations:
...
+
+List of services that will be enabled for users with `manageSystems` permissions and Super Priviliged Mode on:
```
Comment (rkratky) [inline src/main/pages/user-management/permissions.adoc]:
```suggestion
List of services that are enabled for users with `manageSystems` permissions and with super-privileged mode on:

```

File: src/main/pages/user-management/permissions.adoc [hunk 27]
```diff
@@ -73,14 +73,48 @@ The following permissions are applicable to organizations:
...
+[cols=",,",options="header",]
+|===
+|Path | HTTP Method | Description
+|/workspace/namespace/{namespace:.*} |GET |Get all workspaces for given namespace.
```
Comment (rkratky) [inline src/main/pages/user-management/permissions.adoc]:
```suggestion
|`/workspace/namespace/{namespace:.*}` |GET |Get all workspaces for given namespace.
```

File: src/main/pages/user-management/permissions.adoc [hunk 28]
```diff
@@ -73,14 +73,48 @@ The following permissions are applicable to organizations:
...
+|/workspace/{id} |DELETE |Stop workspace
```
Comment (rkratky) [inline src/main/pages/user-management/permissions.adoc]:
```suggestion
|`/workspace/{id}` |DELETE |Stop workspace
```

File: src/main/pages/user-management/permissions.adoc [hunk 29]
```diff
@@ -73,14 +73,48 @@ The following permissions are applicable to organizations:
...
+|/workspace/{key:.*} |GET | Get workspace by key
```
Comment (rkratky) [inline src/main/pages/user-management/permissions.adoc]:
```suggestion
|`/workspace/{key:.*}` |GET | Get workspace by key
```

File: src/main/pages/user-management/permissions.adoc [hunk 30]
```diff
@@ -73,14 +73,48 @@ The following permissions are applicable to organizations:
...
+|/organization/resource/{suborganizationId}/cap |GET |Get resource cap for given organization
```
Comment (rkratky) [inline src/main/pages/user-management/permissions.adoc]:
```suggestion
|`/organization/resource/{suborganizationId}/cap` |GET |Get resource cap for given organization
```

File: src/main/pages/user-management/permissions.adoc [hunk 31]
```diff
@@ -73,14 +73,48 @@ The following permissions are applicable to organizations:
...
+|/organization/resource/{suborganizationId}/cap |POST |Set resource cap for given organization
```
Comment (rkratky) [inline src/main/pages/user-management/permissions.adoc]:
```suggestion
|`/organization/resource/{suborganizationId}/cap` |POST |Set resource cap for given organization
```

File: src/main/pages/user-management/permissions.adoc [hunk 32]
```diff
@@ -73,14 +73,48 @@ The following permissions are applicable to organizations:
...
+|/organization/{parent}/organizations |GET | Get child organizations
```
Comment (rkratky) [inline src/main/pages/user-management/permissions.adoc:118]:
```suggestion
|`/organization/{parent}/organizations` |GET | Get child organizations
```

File: src/main/pages/user-management/permissions.adoc [hunk 33]
```diff
@@ -73,14 +73,48 @@ The following permissions are applicable to organizations:
...
+|/organization |GET | Get user's organizations
```
Comment (rkratky) [inline src/main/pages/user-management/permissions.adoc:119]:
```suggestion
|`/organization` |GET | Get user's organizations
```

File: src/main/pages/workspace-admin/installers.adoc [hunk 34]
```diff
@@ -46,3 +46,18 @@ image::workspaces/installers.png[]
...
 As installer scripts may expect that a user in the container/pod is in `sudoers` .e. have privileges, some of them may fail due to `permission denied` issues. This is the case with OpenShift, where a pod is run by an arbitrary user with no permissions to use sudo, and/or access/modify resources on the file system. In most cases, though, this problem can be solved by rebuilding the base image so that it already has all of the dependencies for a particular agents that an installer activates. Another possible issue might be with permissions to files and folders. For example, an installer may need to write to https://github.com/eclipse/che-dockerfiles/blob/master/recipes/stack-base/centos/Dockerfile#L45-L57[user home directory].
+
+== Installer Registry and REST API
+
+Che installers are stored at Installer Registry. They can be viewed and edited through the Installer Registry REST API:
+
+[cols=",,",options="header",]
+|===
+|Path | HTTP Method | Description
+|/installer |GET | Get installers
```
Comment (rkratky) [inline src/main/pages/workspace-admin/installers.adoc:57]:
```suggestion
|`/installer` |GET | Get installers
```

File: src/main/pages/workspace-admin/installers.adoc [hunk 35]
```diff
@@ -46,3 +46,18 @@ image::workspaces/installers.png[]
...
+|/installer/{id}/version |GET | Get versions of installers by given id
```
Comment (rkratky) [inline src/main/pages/workspace-admin/installers.adoc:58]:
```suggestion
|`/installer/{id}/version` |GET | Get versions of installers by given id
```

File: src/main/pages/workspace-admin/installers.adoc [hunk 36]
```diff
@@ -46,3 +46,18 @@ image::workspaces/installers.png[]
...
+|/installer/orders |GET | Get installers, ordered by their dependencies
```
Comment (rkratky) [inline src/main/pages/workspace-admin/installers.adoc:59]:
```suggestion
|`/installer/orders` |GET | Get installers, ordered by their dependencies
```

File: src/main/pages/workspace-admin/installers.adoc [hunk 37]
```diff
@@ -46,3 +46,18 @@ image::workspaces/installers.png[]
...
+|/installer/ |POST | Add installer to the registry
```
Comment (rkratky) [inline src/main/pages/workspace-admin/installers.adoc:60]:
```suggestion
|`/installer/` |POST | Add installer to the registry
```

File: src/main/pages/workspace-admin/installers.adoc [hunk 38]
```diff
@@ -46,3 +46,18 @@ image::workspaces/installers.png[]
...
+|/installer/{key} |PUT | Update installer in the registry
```
Comment (rkratky) [inline src/main/pages/workspace-admin/installers.adoc:61]:
```suggestion
|`/installer/{key}` |PUT | Update installer in the registry
```

File: src/main/pages/workspace-admin/installers.adoc [hunk 39]
```diff
@@ -46,3 +46,18 @@ image::workspaces/installers.png[]
...
+|/installer/{key} |DELETE | Remove installer from the registry
```
Comment (rkratky) [inline src/main/pages/workspace-admin/installers.adoc]:
```suggestion
|`/installer/{key}` |DELETE | Remove installer from the registry
```

File: src/main/pages/developer-guide/logging.adoc [hunk 40]
```diff
@@ -87,3 +87,24 @@ for example
 ----
 CHE_LOGGER_CONFIG=org.eclipse.che=DEBUG,org.eclipse.che.api.installer.server.impl.LocalInstallerRegistry=OFF 
 ----
+
+Che Server also has an option to view and edit logger configuration during runtime through following REST services:
+
+[cols=",,",options="header",]
+|===
+|GET | logger/ - | get list of all loggers
```
Comment (rkratky) [inline src/main/pages/developer-guide/logging.adoc]:
```suggestion
|GET | `logger/ -` | get list of all loggers
```

File: src/main/pages/developer-guide/logging.adoc [hunk 41]
```diff
@@ -87,3 +87,24 @@ for example
...
+|GET | logger/{name} | get info about particular logger
```
Comment (rkratky) [inline src/main/pages/developer-guide/logging.adoc]:
```suggestion
|GET | `logger/{name}` | get info about particular logger
```

File: src/main/pages/developer-guide/logging.adoc [hunk 42]
```diff
@@ -87,3 +87,24 @@ for example
...
+|POST | logger/{name} | update logger configuration
```
Comment (rkratky) [inline src/main/pages/developer-guide/logging.adoc]:
```suggestion
|POST | `logger/{name}` | update logger configuration
```

File: src/main/pages/developer-guide/logging.adoc [hunk 43]
```diff
@@ -87,3 +87,24 @@ for example
...
+|PUT | logger/{name} | create logger configuration
```
Comment (rkratky) [inline src/main/pages/developer-guide/logging.adoc]:
```suggestion
|PUT | `logger/{name}` | create logger configuration
```

**Review by rkratky — CHANGES_REQUESTED** [review]

File: src/main/pages/user-management/permissions.adoc (hunk 24 above)
Comment (mkuznyetsov) [reply src/main/pages/user-management/permissions.adoc]: A super-privileged mode enables `manageSystem` permission with access to additional services.

Comment (rkratky) [reply src/main/pages/user-management/permissions.adoc]: Thanks for the clarification. I suggest to use:

```suggestion
Super-privileged mode can be enabled, which in turn enables the `manageSystem` permission with access to additional services.
```

**Review by rkratky — APPROVED** [review]

**Pull request merged**